scripts/cibuild

```diff
@@ -3,20 +3,20 @@
 set -e
 
 if [[ -n "${CI_DEBUG}" ]]; then
-    set -x
```

Comment:
None of the changes in this file are meaningful. There are so many other lint changes that I figured I'd `shfmt` this file too.